src/licensedcode/detection.py

```diff
@@ -890,11 +954,20 @@ def dict_fields(attr, value):
 
             return True
 
-        return attr.asdict(self, filter=dict_fields)
+        detection_mapping = attr.asdict(self, filter=dict_fields)
+        detection_mapping["sample_matches"] = [
```

Comment:
What about calling this `reference_matches` as sample seem to convey that these are not so important?

Reply:
this is much better, changed this to reference_matches, thanks.